contrib/opencensus-ext-django/opencensus/ext/django/middleware.py

```diff
@@ -99,7 +99,11 @@ def _set_django_attributes(span, request):
         return
 
     user_id = django_user.pk
-    user_name = django_user.get_username()
+    try:
+        user_name = django_user.get_username()
+    except Exception:
```

Comment:
I think it'd be better checking the Django version or catching a more specific error then a general exception?

Reply:
Good point. Changed this